weapon-config.inc

```diff
@@ -4134,6 +4138,7 @@ static UpdateHealthBar(playerid, bool:force = false)
 
 static SetHealthBarVisible(playerid, bool:toggle)
 {
+	#if WC_VISIBLE_HP_BAR == true
```

Comment:
I don't really understand why you did this, would it have not been better to make a publicly accessible function to toggle this, instead of using a macro?

Reply:
So many ppls want to be with macro thats idea and easy way, i writing this on mobile (the code) so i think that can works without problems, and i can improve it in function but, i mean always can better for fast solution is that.

Reply:
What I am trying to get at is, this seems like a temp fix for a bigger issue and this method only effects a small percentage of people who use this library.

If people want to lock the healthbar so it can never be hidden, then I would prefer you added a way to instead lock the current healthbar to a state that the player can control. This is a much acceptable approach, over trying to do a quick fix. This kinds of things will lead to more problems down the line, and it's not acceptable.

Please considered rethinking your idea, and adding a more acceptable approach that achieves the same goal. If you ask me questions on Discord, I can help you achieve this.

Reply:
Ye i think if set to false this, all working cool but, this will hide a bar but other things workin, what u think? i think this can work, but yea must to be better

Reply:
His point was a function is better than a macro, because then people can enable it and disable it at runtime.